core/CometsLayout.m

```diff
@@ -30,6 +30,9 @@
         initial_pop = [0];%models by x by y, default 0. Units = grams.
         external_rxns = table();
         external_rxn_mets = table();
+        periodic_media_mode = 'none';
+        global_periodic_media = {}; % met by 5 cell array

```

Comment:
update comment to say what each of the five columns is. I'd like to keep this section as the first place to look for all the contents of the matrices

Reply:
Done